README.md

```diff
@@ -1,24 +1,73 @@
+  - Ruby (see .ruby-version)
+  - Rails 7.0+
+  - MySQL 8+  
+  - Redis 6.2+
 
-Things you may want to cover:
+## Template Notes
+After creating your new project, you will need to touch some files to customize the new environment. Those files are:  
 
-* Ruby version
+- config/deploy/* to provide the host names for each deployment target.
+- config/deploy.rb to update the project name & other deployment settings.
+- Run `EDITOR=vim rails credentials:edit` to initialize the Rails encrypted credentials files. See config/credentials.example.yml for some of the environment variables that you should copy to 
+config/credentials.yml.enc
+- config/database.yml to reflect the proper database name and other settings. There is an example in `config/database.example.yml`
+- Look at `config/initializers/site_information.rb` and make changes as needed to the constant values shown there.
+- Perform a 'find all' to look for 'fmaprivacy' and 'fmadata'. Modify those values as needed.
+- You will also need to go through the views to remove/change some of the current boiler plate, logo, etc.
+- This template is configured for the free version of Sidekiq. Review the Gemfile and config/routes.rb files if you are using Sidekiq Pro.
+- Change the name of the application in `config/application.rb`. Change FMAdataTemplate to something meaningful.
 
-* System dependencies
+NOTE: If you find other files that need to be touched after creating a new project please come back here and update the documentation!
 
-* Configuration
+## Configuration and installation
 
-* Database creation
+### Credentials 
 
-* Database initialization
+See Keybase for master.key for this project.
+To generate credentials, run:
+  - in atom, use `EDITOR="atom --wait" rails credentials:edit`
+  - in vim use `EDITOR=vim rails credentials:edit`
+Enter your keys, save and exit. See config/credentials.example.yml for examples.
 
-* How to run the test suite
+Set up locking strategy during configuration (https://github.com/plataformatec/devise/wiki/How-To:-Add-:lockable-to-Users).
 
-* Services (job queues, cache servers, search engines, etc.)
+### Gems
 
-* Deployment instructions
+Run `bundle install` command to install Rails and all other Gem dependencies. 
 
-* ...
+### Database creation
+
+MySQL 8 is the recommended version to run on your machine. Once you have MySQL running, run:
+  `cp config/database.example.yml config/database.yml`
+and configure your config/database.yml to your machine's MySQL configuration. Minor edits should only be necessary.
+
+Run rake db:setup to create and migrate the database from schema (db/schema.rb).
+
+### Attribute Encryption
+Attribute encryption on models is required for any attribute that contains personal information or other sensitive data. The default solution is the attr_encrypted gem.
+
+## Other dependencies 
+
+### Redis
+in order for Sidekiq to work it requires redis version of at least 6.2 (but since it's deprecated, recommended version is 7)
```

Comment:
I'd remove v7 mention. It's the newest version at the moment, but it will also be deprecated in the future. Redis version required for sidekiq is 6.2+, so I'd leave it at this.